### ISSUE TITLE
Binary files are not deleted

### ISSUE THREAD

**Issue by rolandstarke** [sheershoff/node-cache-manager-fs-binary#6]
Hi, first of all thanks for the awesome library.

I have got a problem. Somehow in my folder the files pile up. It looks like the binary files don't get deleted.

You can recreate the issue with the following code:

```javascript
// node cachemanager
var cacheManager = require('cache-manager');
// storage for the cachemanager
var fsStore = require('cache-manager-fs-binary');
// initialize caching on disk
var diskCache = cacheManager.caching({
    store: fsStore,
    options: {
        path: __dirname,
        fillcallback: fillcallback
    }
});


function fillcallback() {
    diskCache.set('cacheKey', {
        binary: {smallFile: Buffer.alloc(100, 128)}
    });

    setTimeout(function () {
        diskCache.del('cacheKey', function (err) {
            if (err) throw err;
            console.log('deletion done');
        });
    }, 1000);
}
```

In my folder the file `cache_6e8ec1b4-e3b7-4a3d-a126-0365dc6b05d9_smallFile.bin
` is then left over. Each time I run the program a new file gets is added.

Best regards
Roland

**Comment by rolandstarke** [comment]
I see its already fixed with #2 . would be nice if you could upload the newest version to npm :)

**Comment by sheershoff** [comment]
Did that. )
